src/actions/eventTypeActions.js

```diff
@@ -4,13 +4,18 @@ export const TRY_GET_EVENT_TYPE = 'TRY_GET_EVENT_TYPE'
 export const GET_EVENT_TYPE_SUCCESS = 'GET_EVENT_TYPE_SUCCESS'
 export const GET_EVENT_TYPE_FAILURE = 'GET_EVENT_TYPE_FAILURE'
 
+export const TRY_GET_EVENT_TYPES = 'TRY_GET_EVENT_TYPES'
+export const GET_EVENT_TYPES_SUCCESS = 'GET_EVENT_TYPES_SUCCESS'
+export const GET_EVENT_TYPES_FAILURE = 'GET_EVENT_TYPES_FAILURE'
 
 export const fetchEventType = (eventTypeId) => reduxBackedPromise(
     ['eventTypes/' + eventTypeId],
     getEventTypeActionSet(eventTypeId)
 )
 
-
+export const fetchEventTypes = (history) => reduxBackedPromise(
+    ['eventTypes/'], getEventTypesActionSet(history)
```

Comment:
No longer needs history; getEventTypesActionSet doesn't need to be a function anymore.

Reply:
Fixed